vercel.json

```diff
@@ -12,8 +12,8 @@
       }
     },
     {
-      "src": "/favicon.ico",
-      "dest": "/favicon.ico"
+      "src": "/(.*)\.ico",
```

Comment:
My bad, we probably need `/(.*).ico` instead of `/(.*)\.ico`.

Reply:
No problem, I'll push the fix now.